aten/src/ATen/test/atest.cpp

```diff
@@ -9,7 +9,7 @@ using namespace std;
 using namespace at;
 
 void trace() {
-  Tensor foo = rand(CPU(kFloat), {12,12});
+  Tensor foo = rand({12,12});
```

Comment:
I think it would be worth it to add backwards compatibility functions to TensorOption for
`ones, zeros, rand, tensor`. That way `at::CPU(at::kFloat).ones()` still works, for example. These can be written manually and forward to the `at::ones` versions. If you look at these tests it is nearly 25% of the lines that need modification, and I worry how much we will break our few C++ users. Those 4 functions are one liners and handle the majority of cases.

Reply:
Good idea